Radzen.Blazor/RadzenAlert.razor

```diff
@@ -1,7 +1,8 @@
 ﻿@inherits RadzenComponentWithChildren
+<div aria-live="polite">
```

Comment:
Changing the component rendering is a breaking change.

Reply:
How could we solve this? Problem is the live region must be present for the screen reader to read the modification. Is there possiblity to add it to the next major update?

Reply:
You can easily add that div whenever you use RadzenAlert.

Reply:
Yeah, got it. I'll remove it. Would the Changes of the RadzenNotification accepted alone?

Reply:
I think the changes to RadzenNotification are not a breaking change. You can update the pull request.

Reply:
done